pkg/driver/attacher.go

```diff
@@ -41,7 +41,7 @@ func multipath(args ...string) (string, error) {
 	output, err := cmd.Output()
 
 	if ctx.Err() == context.DeadlineExceeded {
-		glog.V(5).Infof("multipath (%s) timed out after %v", strings.Join(args, " "), multipathTimeout)
+		log.WithFields(log.Fields{"timeout": multipathTimeout, "args": strings.Join(args, " ")}).Infof("multipath timed out")
```

Comment:
This one actually should be Info, because it's "expected". Calling `multipath` from inside a container is flaky, at least with the versions of multupath/multipathd currently in use.  multipath is deadlocking before returning, so there's not much to do except abort.